website/docs/r/vm.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vm"
+sidebar_current: "docs-vcd-resource-vm"
+description: |-
+  Provides a vCloud Director VM resource. This can be used to create, modify, and delete standalone VMs.
+---
+
+# vcd\_vm
+
+Provides a VMware Cloud Director standalone VM resource. This can be used to create, modify, and delete standalone VMs.
+
+Supported in provider *v3.2+*
+
+## Example Usage
+
+```hcl
+resource "vcd_vm" "TestVm" {
+  name          = "TestVm"
+
+  catalog_name  = "cat-datacloud"
+  template_name = "photon-hw11"
+  cpus          = 2
+  memory        = 2048
+
+  network {
+    name               = "net-datacloud-r"
+    type               = "org"
+    ip_allocation_mode = "POOL"
+  }
+}
+```
+
+## Arguments and attributes reference
+
+This resource provides all arguments and attributes available for `vcd_vapp_vm`, with the only difference that the
+`vapp_name` is not required.
```

Comment:
It's not not required, it does not exist :) So, we should say something like:

```
with the only difference that there is no `vapp_name`.
```

Reply:
changed